### DISCUSSION TITLE
Could you add a way to manually translate the text

### DISCUSSION THREAD

**Discussion by Abelkrijgtalles** [ThioJoe/Auto-Synced-Translated-Dubs#14] [Ideas]
I don't like to add a credit card to google cloud, so could you maybe add a way to translate the text manually.

This could be a way to just do it manually through Google Translate or do something like Dutch to English.

**Comment by sofiadparamo** [comment]
I think you can just comment out line `338` from `main.py` 
```python
137|    # Translate
138|    individualLanguageSubsDict = translate_dictionary(individualLanguageSubsDict, langDict, skipTranslation=skipTranslation)
```
and put your translated subtitles inside a folder `output`. The name of the file should be `video - <Language> - <lang short code>.srt`, for example, `video - Spanish - es.srt`.

Note that this is assuming you're translating using the same length for phrases as your original subtitles, if you change the structure or order of the phrases you may get weird results. Also, I'm assuming the settings for video file name are unchanged (video.mp4 and subtitles.srt)

**Comment by kaswardy** [comment]
I tried this but it gave me an error. Is there anything else that I should be doing?

**Comment by Pythone** [comment]
> I don't like to add a credit card to google cloud, so could you maybe add a way to translate the text manually.
> 
> 
> 
> This could be a way to just do it manually through Google Translate or do something like Dutch to English.


That is why I created discussion #59
